assets/scss/components/_summary-card.scss

```diff
@@ -5,14 +5,6 @@
   padding: 16px 0 0 16px;
 }
 
-.app-summary-card__title-wrapper--green {
```

Comment:
From day 1 we (devs) always questioned why this section was a different colour, and why that colour was green.
Our concern was that is was not accessible, in that if using a different colour was meant to convey some kind of meaning or significance, then using colour alone was not enough. We also questioned why green, as green might suggest "good" or "complete" or "better than something that is not green" etc.

![Screenshot 2024-05-14 at 08 48 39](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/94835226/52086013-a09a-4f15-a87c-25768c5a8a12)

I'm glad this has been changed 👍